packages/@lightningjs/ui-components/src/components/Tile/Tile.js

```diff
@@ -568,14 +568,7 @@ export default class Tile extends Surface {
   }
 
   _getMetadataLocation() {
```

Comment:
Faster not to modify style here. Not sure there would be any case it's preferable to do it this way though?

Reply:
it may have been to keep styles in sync with props but im not sure if that's correct/relevant anymore?

Reply:
@erautenberg:
> So this makes sure that if the user sets it on the component and not nested in styles, it updates the styles. And we have to do this.style = {} instead of this.style.prop = '' because there are no individual setters for properties nested in style that would trigger a new update lifecycle, so you have to set on the style itself

resolving this per our conversation